src/main/webapp/aboutus.html

```diff
@@ -41,17 +41,13 @@ <h2>Teammate B Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate C Name</h2>
-    <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
-    </ul>
-    <h2>Teammate D Name (delete if you team only has 3 students) </h2>
-    <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+    <link href="https://fonts.googleapis.com/css?family=Anton|Bungee+Shade&display=swap" rel="stylesheet">
+    <h2 style="font-family: 'Bungee Shade', cursive; letter-spacing:2px">Breelyn</h2>
```

Comment:
In general, try to avoid inline styling. Put the styles in main.css instead, and then apply them to these elements via a class or an ID. Inline styling is hard to maintain and scale, which is why CSS was invented.

Reply:
Okay will do! Sorry I wasn't sure if we were supposed to modify the CSS file for this assignment! Thank you for your help!